src/Components/WebDevCourseCard/WebDevCourseCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styles from "./WebDevCourseCard.module.scss";
+import { devCourseDataType } from "@/types/devCourseData";
```

Comment:
Bu componente ihtiyacımız yok. Bunun yerine mevcut olan CourseCard componentimiz ile course'ları render edebiliriz. Bu sayede react component mantığını gerçek projeye göre daha iyi bir şekilde uyarlamış olacağız.
- Kullanım örneği için diğer componentlerde <CourseCard />'ın nasıl kullanıldığına bakılabilir.